.github/actions/entrypoint.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
-runner_uid=$(id -u)
-runner_gid=$(id -g)
-
+runner_uid=$(ls -lnd . | awk '{ print $3; }')
```

Comment:
did `id -u` not work? wrong user id in container? :)

Reply:
Yeah, it somehow did not work :D So I reverted to the older, working way.